test/extended/dns/dns.go

```diff
@@ -114,6 +118,17 @@ func digForARecords(records map[string][]string, expect sets.String) string {
 	return probeCmd
 }
 
+func digForAAAARecords(records map[string][]string, expect sets.String) string {
+	var probeCmd string
+	fileNamePrefix := "test"
```

Comment:
if `fileNamePrefix` never changes, just hard code it in the base string of the `Sprintf`

Reply:
This is how the other `digForFooRecords` functions are laid out. I think I'll leave this as is for consistency sake.